recipes/morphometrics/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "morphometrics" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/morphometrics-{{ version }}.tar.gz
+  sha256: 096ed657ef55aa1ff61e1fedfcde4ab882fcf115dbe488944da27194bc8e652a
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [py==37]
```

Comment:
@goanpeca, `morphometrics` isn't compatible with python 3.7 I just realized I forgot to update the `setup.cfg` metadata. If I do so, and cut a new release on pypi will this remove this conditional?

Reply:
Hi @kevinyamauchi, this will still be needed, so no need to do any updates.

Reply:
Thanks!